src/storybook/helpers.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FunctionComponent } from 'react'
+
```

Comment:
Only for cleaning up, I would remove the lines or is there a reason for the spacing?

Reply:
```suggestion
```